aip/_journals.tab

```diff
@@ -1,4 +1,6 @@
 Title	ISSN	eISSN	Field	Field2
+Journal of Vacuum Science &amp; Technology A: Vacuum, Surfaces, and Films 0734-2101 x physics chemistry
+Journal of Vacuum Science &amp; Technology B: Microelectronics and Nanometer Structures 1520-8567 x physics chemistry
```

Comment:
@damnation333:

1. You don't have to escape characters like "&" here.
2. Please keep these files alphabetically organized if they are.
3. It looks like you didn't use tabs to separate the fields.